bench/workgen/workgen.cxx

```diff
@@ -2255,11 +2385,17 @@ int WorkloadRunner::run_all(WT_CONNECTION *conn) {
     }
 
     // Wait for the time increment thread
-    if (options->oldest_timestamp_lag > 0 || options->stable_timestamp_lag > 0) {
+    if ( runnerConnection != NULL) {
```

Comment:
remove extra space between ( and "runnerConnection"

Reply:
Fixed